test/Microsoft.ML.Tests/Scenarios/Api/AspirationalExamples.cs

```diff
@@ -29,25 +29,23 @@ public class IrisExample
         public void FirstExperienceWithML()
         {
             // Load the data into the system.
-            string dataPath = "iris-data.txt";
-            var data = TextReader.FitAndRead(env, dataPath, row => (
-                Label: row.ReadString(0),
-                SepalWidth: row.ReadFloat(1),
-                SepalLength: row.ReadFloat(2),
-                PetalWidth: row.ReadFloat(3),
-                PetalLength: row.ReadFloat(4)));
+            string dataPath = "iris.txt";
+            var data = TextReader.CreateReader(env, dataPath, row => (
```

Comment:
This example is inconsistent with the others.  Here we say `var data = TextReader.CreateReader`, and we never call "Read" on the thing returned from `CreateReader`. We just pass it into `Fit` below.

But in other examples we say `var reader = TextLoader.CreateReader` and later call `var data = reader.Read(dataPath);`.